activities/models/fan_out.py

```diff
@@ -17,11 +17,15 @@ async def handle_new(cls, instance: "FanOut"):
         """
         Sends the fan-out to the right inbox.
         """
+        LOCAL_IDENTITY = True
+        REMOTE_IDENTITY = False
+
         fan_out = await instance.afetch_full()
-        # Handle Posts
-        if fan_out.type == FanOut.Types.post:
-            post = await fan_out.subject_post.afetch_full()
-            if fan_out.identity.local:
+
+        match (fan_out.type, fan_out.identity.local):
+            # Handle creating/updating local posts
+            case (FanOut.Types.post | FanOut.Types.post_edited, LOCAL_IDENTITY):
+                post = await fan_out.subject_post.afetch_full()
```

Comment:
This treats new and updates identically without caring to clean up target differences between original and updated. Should it?

Reply:
That's kind of how the rest of the code treats it, so I think we're fine. The only thing I'd care about is marking that a post has been edited, which we can retrieve from the post model directly.